types/options.go

```diff
@@ -220,9 +220,9 @@ type StoreOptions struct {
 	// GraphRoot is the filesystem path under which we will store the
 	// contents of layers, images, and containers.
 	GraphRoot string `json:"root,omitempty"`
-	// Image Store is the location of image store which is seperated from the
+	// Image Store is the location of image store which is separated from the
 	// container store. Usually this is not recommended unless users wants
-	// seperate store for image and containers.
+	// separate store for image and containers.
```

Comment:
I think this needs more than just spelling fixes? The first correction should IMO also be "separate", not separated. And the second instance is just repetition, not really helpful. Is there a way to phrase that such that it would explain the _why_, not the what?

Reply:
Ok I took a stab at improving the description of imagestore in all locations.